system/jlib/jstring.cpp

```diff
@@ -1281,6 +1286,20 @@ StringAttr::StringAttr(const StringAttr & src)
     set(src.get());
 }
 
+StringAttr::StringAttr(StringAttr && src)
+{
+    text = src.text;
+    src.text = nullptr;
+}
+
+StringAttr& StringAttr::operator = (StringAttr && from)
+{
+    free(text);
```

Comment:
It is ok.  More conventional (and safer) would be 

temp = text
text = from.text
from.text = temp;